index.js

```diff
@@ -108,11 +108,23 @@ program
 
     const site = new Site(rootFolder, outputFolder);
 
+    const addHandler = (filePath) => {
+      logger.info(`Reload for file add: ${filePath}`);
+      Promise.resolve('').then(() => {
+        if (fsUtil.isMarkdown(filePath) || fsUtil.isHtml(filePath)) {
```

Comment:
Seems like we use these two conditions multiple times, consider extra it to a function in `fsUtil`: `isSourceFile()`